src/components/LandingPage/LandingPage.scss

```diff
@@ -0,0 +1,142 @@
+@import '../../styles/variables';
+
+.landing-page {
+  display: grid;
+  grid-template-columns: auto;
+  min-height: 100vh;
+  background-color: $landing-page-background;
+}
+
+.landing-page__hero-section {
+  display: grid;
+  grid-gap: 20px;
+  grid-template-columns: 1fr auto 1fr;
+  overflow: hidden;
+  position: relative;
+  color: $white;
+  background: linear-gradient(180deg, #1f6cbf 0%, rgba(255, 255, 255, 0) 100%), #007bff;
+  padding: 84px 84px 10vw;
+  min-height: 450px;
+  max-height: 60vh;
+}
+
+.landing-page__hero-section::after {
+  content: '';
+  position: absolute;
+  // Otherwise blue background shows one pixel underneath the wave.
+  bottom: -2px;
+  width: 102%;
+  background-image: url('./wave.svg');
+  background-size: cover;
+  background-repeat: no-repeat;
+  height: 120px;
+  pointer-events: none;
+}
+
+@media (max-width: 1685px) {
+  .landing-page__hero-section::after {
+    background-size: 103%;
+    background-position: bottom;
+  }
+}
+
+.landing-page__heading {
+  font-weight: 900;
+  font-size: 3.5rem;
+  letter-spacing: 1px;
+  line-height: 1;
+}
+
+.landing-page__sub-heading {
+  text-align: left;
+}
+
+.landing-page__simulate-link {
+  align-self: end;
+  justify-self: center;
+  color: $black;
+  border-radius: 500px;
+  padding: 18px 60px;
+  font-weight: 700;
+  font-size: 16px;
+  background-color: $white;
+}
+.landing-page__simulate-link:hover,
+.landing-page__simulate-link:focus {
+  text-decoration: none;
+  color: $black;
+  transform: scale(1.1);
+}
+.landing-page__simulate-link:active {
+  transform: scale(0.99);
+}
+
+.landing-page__wave {
+  pointer-events: none;
+  background: transparent;
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  transform: scaleX(6);
+  color: $landing-page-background;
+}
+
+.landing-page__promo-image {
+  position: absolute;
+  top: 42px;
+  right: 42px;
+  width: 600px;
+}
+
+@media (max-width: 1645px) {
+  .landing-page__promo-image {
+    width: 450px;
+  }
+}
+
+.landing-page__content-section {
+  padding: 0 84px 20px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  // put on top of svg wave
+  z-index: 2;
+  // move closer to the wave. Based on screen width.
+  // This prevents excessive white space.
+  margin-top: -3vw;
+}
+
+.landing-page__content-section-text {
+  flex: 1;
+  max-width: 400px;
+}
+
+.skip-landing-page-label {
+  font-size: 12px;
+  margin-left: 0.5rem;
+}
+
+@media (max-width: 768px) {
+  .landing-page__hero-section {
+    grid-template-columns: auto;
+    grid-template-rows: auto;
+    padding: 40px 40px 10vw 40px;
+    min-height: 350px;
+  }
+
+  .landing-page__simulate-button {
+    margin: 30px 0 40px;
+  }
+
+  .landing-page__promo-image {
+    position: unset;
+    display: block;
+    width: 100%;
+    margin-bottom: 20px;
+  }
+
+  .landing-page__content-section {
+    padding: 20px 40px;
+  }
+}
```

Comment:
I'm not happy with the state of this CSS. I might spend some more time to get it into better shape. e.g. not use grid, consolidate breakpoints, contain weirdness due to the svg wave better, less magic numbers, use more variables, etc.